docs/samples/browser-plugin-meetings/app.js

```diff
@@ -367,16 +454,17 @@ function joinMeeting({withMedia, withDevice} = {withMedia: false, withDevice: fa
     getMediaDevices();
   }
 
-
   const joinOptions = {
     pin: meetingsJoinPinElm.value,
+    captcha: meetingsJoinCaptchaElm.value,
```

Comment:
is this really needed? I don't see captcha being read from joinOptions when I follow the meeting.join() implementation

Reply:
Let me check it and remove. Thanks.